default/python/AI/AIDependencies.py

```diff
@@ -102,30 +103,36 @@ class Tags:
 POP_SIZE_MOD_MAP_MODIFIED_BY_SPECIES = {
     # "Effect": [uninhab, hostile, poor, adequate, good]
     "environment_bonus": [0, -4, -2, 0, 3],
-    "GRO_SYMBIOTIC_BIO": [0,  0,  1, 1, 1],
```

Comment:
extra whitespace was clearly intentional... you prefer to remove?

Reply:
I am not sure that we lost a lot of readability.  I don't want to make an exclusion for a couple of variables.

**The Zen of Python**
> Special cases aren't special enough to break the rules.

Reply:
as you prefer for Python code...